plugins/obs-qsv11/obs-qsv11.c

```diff
@@ -635,8 +635,9 @@ static void update_params(struct obs_qsv *obsqsv, obs_data_t *settings)
 		obsqsv->params.WhitePointX = 15635;
 		obsqsv->params.WhitePointY = 16450;
 		obsqsv->params.MaxDisplayMasteringLuminance =
-			hdr_nominal_peak_level * 10000;
-		obsqsv->params.MinDisplayMasteringLuminance = 0;
+			hdr_nominal_peak_level;
+		obsqsv->params.MinDisplayMasteringLuminance =
+			obsqsv->codec == QSV_CODEC_AV1 ? 0 : 1;
```

Comment:
Did you check the metadata? Don't just trust a header comment.

Reply:
Hadn't been able to check yet, hence I left it as a draft. Double checked it now and the header is a lie! Reverted this change now.

Also turns out https://github.com/obsproject/obs-studio/commit/4c6e2a6722de4ed923cf29bc0bf65ddd8f45a2cf broke QSV on my NVIDIA+Intel system so that's fun...